From d0587b61dd3fabcd58711ce781f7fa4532b569b0 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 11 Sep 2014 16:35:12 +0200
Subject: [PATCH 12/60] prefs: disable "Use a web service to help resolve
 navigation errors"

This goes towards reducing promiscuous outbound requests to
some central services.

Impl. details:
Set alternate_error_pages to be disabled by default.
---
 chrome/browser/ui/navigation_correction_tab_observer.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/ui/navigation_correction_tab_observer.cc b/chrome/browser/ui/navigation_correction_tab_observer.cc
index deb42bccd1c5..cd0576dfa621 100644
--- a/chrome/browser/ui/navigation_correction_tab_observer.cc
+++ b/chrome/browser/ui/navigation_correction_tab_observer.cc
@@ -56,7 +56,7 @@ NavigationCorrectionTabObserver::~NavigationCorrectionTabObserver() {
 void NavigationCorrectionTabObserver::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* prefs) {
   prefs->RegisterBooleanPref(prefs::kAlternateErrorPagesEnabled,
-                             true,
+                             false,
                              user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 }
 
-- 
2.19.1

